tests/main/cgroup-tracking-failure/task.yaml

```diff
@@ -0,0 +1,198 @@
+summary: cases where tracking applications and hooks with cgroups does not work
+details: |
+    Cgroup-based application process tracking is implemented using the systemd
+    DBus API org.freedesktop.systemd1.Manager.StartTransientUnit. While exactly
+    what we need in spirit, in practice there are limitations that have been
+    overcome only starting with systemd version 238, which can forward a
+    request from systemd --user to systemd pid one, to perform some cgroup
+    transitions that the kernel normally does not allow.
+systems:
+    - ubuntu-14.04-*
+    - ubuntu-16.04-*
+    - ubuntu-18.04-*
+    - ubuntu-20.04-*
```

Comment:
Maybe group 20.04 with core-20 together with a comment that `# used for reference, tracking works correctly` or something along those lines.

Reply:
Done